gammapy/data/filters.py

```diff
@@ -48,6 +48,11 @@ def __init__(self, time_filter=None, event_filters=None):
         self.time_filter = time_filter
         self.event_filters = event_filters or []
 
+    @property
+    def livetime_fraction(self):
```

Comment:
Is this a good name? Off hand, it seems like a `livetime_duration/observation_duration`... like `deadtime_fraction`
Also, the doc string `resulting from event_filters` is a bit confusing.

Reply:
Well, this is really the livetime that you keep in your observation. But I am open to other suggestion !